client/src/context/store.js

```diff
@@ -8,7 +8,7 @@ import {
 } from './types';
 
 const initialState = {
-  token: localStorage.getItem(process.env.REACT_APP_USER_DATA),
+  token: null,
```

Comment:
If we are using cookies, we shouldn't be storing the token, or there is no point of using cookies then

Reply:
Right, so remove the field entirely and change any conditional rendering statements / Redirects to something different than state.token.  Simply using state.isAuthenticated should work, right?